Sources_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -160,6 +163,8 @@ extension ChannelModel {
             .load(for: dto.cid, limit: 25, context: dto.managedObjectContext!)
             .map { $0.asModel() }
         
+        let reads = dto.reads.map(ChannelReadModel<ExtraData>.create(fromDTO:))
+        
```

Comment:
Ideally, you'd do just `dto.reads.map { $0.asModel() }` like above with the messages.

Reply:
Fixed in 62b46eb0b0bef360c33b591d1dbceb93446701e2